src/config/HelpMenuConfiguration.json

```diff
@@ -1,6 +1,6 @@
 {
   "APP_VERSION": null,
-  "COSMOTECH_URL": "https://cosmotech.com",
+  "ORGANIZATION_URL": "https://cosmotech.com",
```

Comment:
We don't need to change it in _HelpMenuConfiguration.vanilla.json_ ? :slightly_smiling_face:

Reply:
You're right, I'll add this too, thank you ! :smile: